docs/docs/integrations/llms/friendli.ipynb

```diff
@@ -40,21 +40,32 @@
    "execution_count": 1,
    "metadata": {},
    "outputs": [],
-   "source": ["import getpass\nimport os\n\nif \"FRIENDLI_TOKEN\" not in os.environ:\n    os.environ[\"FRIENDLI_TOKEN\"] = getpass.getpass(\"Friendi Personal Access Token: \")"]
+   "source": [
+    "import getpass\n",
+    "import os\n",
+    "\n",
+    "if \"FRIENDLI_TOKEN\" not in os.environ:\n",
+    "    os.environ[\"FRIENDLI_TOKEN\"] = getpass.getpass(\"Friendi Personal Access Token: \")"
+   ]
   },
   {
    "cell_type": "markdown",
    "metadata": {},
    "source": [
-    "You can initialize a Friendli chat model with selecting the model you want to use. The default model is `mixtral-8x7b-instruct-v0-1`. You can check the available models at [docs.friendli.ai](https://docs.periflow.ai/guides/serverless_endpoints/pricing#text-generation-models)."
+    "You can initialize a Friendli chat model with selecting the model you want to use.  \n",
+    "The default model is `meta-llama-3.1-8b-instruct`. You can check the available models at [docs.friendli.ai](https://docs.periflow.ai/guides/serverless_endpoints/pricing#text-generation-models)."
```

Comment:
Could you replace `docs.periflow.ai` or other deprecated urls in this PR together?